pkg/statser/statser_internal.go

```diff
@@ -0,0 +1,100 @@
+package statser
+
+import (
+	"context"
+	"time"
+
+	"github.com/atlassian/gostatsd"
+)
+
+// InternalHandler is an interface to dispatch metrics to.  Exists
+// to break circular dependencies.
+type InternalHandler interface {
+	DispatchMetric(ctx context.Context, m *gostatsd.Metric) error
```

Comment:
I usually omit formal argument names in interface declarations when typed are more than enough to understand what the arguments are.